tests/TestCase/Shell/BakeShellTest.php

```diff
@@ -87,7 +87,7 @@ public function testAllWithModelName()
             ->with('Comments');
 
         $this->Shell->connection = '';
-        $this->Shell->params = [];
+        $this->Shell->params = ['prefix' => 'account'];
```

Comment:
So it is something to check again... This is not the behavior that I was expecting. Tonight I can check it better! Thanks @antograssiot   :+1: